File: Java/RefactoringWorkshop/src/main/java/changepreventors/divergentchange/Customer.java
package changepreventors.divergentchange;

public class Customer {
    private String firstName;
    private String lastName;
    private String title;

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String toXML() {
        return "<customer>" +
                    "<firstName>" + getFirstName() + "</firstName>" +
                    "<lastName>" + getLastName() + "</lastName>" +
                    "<title>" + getTitle() + "</title>" +
                "</customer";
    }

    public String toJSON() {
        return "{" +
                    "firstName: " + getFirstName() + "," +
                    "lastName:" + getLastName() + "," +
                    "title:" + getTitle() +
                "}";
    }


}

File: C#/RefactoringWorkshop/RefactoringWorkshop/ChangePreventors/DivergentChange/Customer.cs
namespace RefactoringWorkshop.ChangePreventors.DivergentChange
{
    public class Customer
    {
        public string FirstName { get; set; }

        public string LastName { get; set; }

        public string Title { get; set; }

        public string ToXml() {
            return "<customer>" +
                       "<firstName>" + FirstName + "</firstName>" +
                       "<lastName>" + LastName + "</lastName>" +
                       "<title>" + Title + "</title>" +
                   "</customer";
        }

        public string ToJson() {
            return "{" +
                       "firstName: " + FirstName + "," +
                       "lastName:" + LastName + "," +
                       "title:" + Title +
                   "}";
        }

    }
}